### ISSUE TITLE
Logout from Amazon issue

### ISSUE THREAD

**Issue by Bharath-Ponamgi** [willblaschko/AlexaAndroid#127]
Hi,
**Problem with Login with Amazon and Logout From Amazon**
While trying to logout from the Amazon Alexa account its getting into the onSuccess callback method but after that when I tried to login using alexaManager.logIn(authorizationCallback);  its calling onSuccess callback method without redirecting to Login page to enter credentials.

 Below is the code that I'm trying for Logout.
AuthorizationManager.signOut(getApplicationContext(), new Listener<Void, AuthError>() {
                    @Override
                    public void onSuccess(Void response) {
                  
                    }
                    @Override
                    public void onError(AuthError authError) {
                        Log.e(TAG, "Error clearing authorization state.", authError);
                    }
                });

Note: In some phones when trying to login its redirecting to other browser apps like chrome and in some phones it is redirecting to webview in the same app. If its in same app's webview it is taking to **Login** Screen of Amazon and if its from Browser apps then its taking to **Signin** Screen of Amazon. These two screens are completely different, which one is correct? I'm mainly using this for Alexa I'm facing the issues in this as well. So if I logged out from amazon account successfully then will Alexa's auth manager will go to logout state or not?

**Comment by renu-parameswaran** [comment]
Hi, I have been facing the same issue but i have a problem with authorization- that it shows "We're sorry!
An error occurred when we tried to process your request. Rest assured, we're already working on the problem and expect to resolve it shortly." Do you know to resolve this?

**Comment by Bharath-Ponamgi** [comment]
Hi Renu, to solve the authorization problem you are facing, make sure of the **Product Id** & **Product DSN**  that you have integrated in the project and in the Amazon console are same. Basically the issue **"We're sorry!
An error occurred when we tried to process your request. Rest assured, we're already working on the problem and expect to resolve it shortly."** will come only when the credentials that we pass are wrong.

**Comment by renu-parameswaran** [comment]
Hi, Thanks for replying, but we are only required to put product id in project right? Where do we get Product DSN from?

**Comment by Bharath-Ponamgi** [comment]
Yes, I think Product DSN is not required for now as the library inbuilt having it. Check the product Id is correct or not.

**Comment by Bharath-Ponamgi** [comment]
By the way the Product DSN is the Amazon ID that will be in Amazon Developers Console -> Alexa -> My Products section

**Comment by renu-parameswaran** [comment]
okay, where am i supposed to include product DSN in the project, I'm still facing the same issue. I rechecked my product id, it seems to be correct.

**Comment by ghost** [comment]
@Bharath-Ponamgi Thank you. It was about product ID. I resolved it.
